src/toolchain.rs

```diff
@@ -629,6 +630,35 @@ impl<'a> Toolchain<'a> {
         }
     }
 
+    fn get_component_suggestion(&self, component: &Component) -> Option<String> {
+        use levenshtein::levenshtein;
+        const MAX_DISTANCE: usize = 3;
```

Comment:
A comment explaining why you chose `3` would be useful, so that it's not just a magical constant.